### DISCUSSION TITLE
Create Anki from full note - Any advice?

### DISCUSSION THREAD

**Discussion by rob-hamill** [ObsidianToAnki/Obsidian_to_Anki#222] [Q&A]
Hi,

I really like the plugin, but am trying to find a way of allowing full notes (.md files) to be created in the cards. This may seem odd, but I want the spaced-repition features of Anki, and have purchased the IOS app in order to review notes. (This is preferable in many ways to a standard markdown editor on the iPad).

**Currently**
Currently I am having to have an additional section at the bottom of notes and then copy all the preceding content into the relevant syntax. 
Example:

Actual note content (containing mix of headings, lists,latex etc)

```
# Anki
TARGET DECK: Engineering
START
Basic
Front: 
<br>
## Title of Note
<br>
Back: 
<br>
Copy of note content (containing mix of headings, lists,latex etc)
<br>
<!--ID: 12345678910-->
END`
``` 

This is less than ideal as:
1. It means copying and pasting content every time there is a modification
2. It reduces the readibility and long-term robustness of the notes

**Possible Solution?**
I would love to be able to create a seperate file for my Anki notes in which I could create a card through embedding the note file. 
Example:

```
TARGET DECK: Engineering
START
Basic
Front: 
<br>
## Title of Note
<br>
Back: 
<br>
![[Note File]]
<br>
<!--ID: 12345678910-->
END` 

TARGET DECK: Engineering
START
Basic
Front: 
<br>
## Title of Note 2
<br>
Back: 
<br>
![[Note File 2 ]]
<br>
<!--ID: 12345678911-->
END` 

```

If anyone has anyone workarounds / solutions / advice please let me know! Really keen to make this smoother.
Thanks!

**Comment by base-I** [comment]
I just write down questions on the front of the card for active recall and then I click the link in the back an answer these questions one by one while referring tho the note

**Reply by rob-hamill** [reply]
Yeah, for my use case it would be really valuable to find a method of displaying the full note (or specified section) which will update etc. 

For this I believe the plugin would need to support exported embedded note files or block references (but this would also require manual block definition to be implemented to obsidian).
